db.js

```diff
@@ -10,3 +10,38 @@ export function dbTrips() {
     },
   ];
 }
+
+export function dummyDestinations() {
+  return [
+    {
+      id: 1,
+      name: "Bangkok",
+      tripId: 1,
+    },
+    {
+      id: 2,
+      name: "Cebu City",
+      tripId: 2,
+    },
+    {
+      id: 3,
+      name: "Chiang Mai",
+      tripId: 1,
+    },
+    {
+      id: 4,
+      name: "Tokio",
+      tripId: "bc9d0d74",
+    },
```

Comment:
is there a reason why the tripIds are not structured in the same way?

Reply:
The first id's were just dummies. I'm going to change them all to the same string structure today. Thanks!